bibliopixel/project/project.py

```diff
@@ -44,10 +45,44 @@ def make_animation(layout, animation, run=None):
     return animation
 
 
-def _make_project(path=None, animation=None, run=None, **kwds):
+def make_remote(layout, remote):
+    OFF_ANIM = {
+        'typename': 'bibliopixel.animation.off.OffAnim'
+    }
+
+    animations = remote.get('animations', None)
+    if not animations or len(animations) == 0:
+        raise ValueError('Must specify `animations` for remote.')
+
+    anim_list = []
+    for anim in animations:
+        run = anim.get('run', {})
+        run['threaded'] = True  # Remote anims must be threaded, maybe a little hacky?
+        anim_obj = dict(anim)
+        anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        anim_obj.pop('run', None)  # no longer need this
+        anim_list.append(anim_obj)
+
+    base_config = dict(remote)
+    base_config.pop('animations', None)
```

Comment:
A few lines down RemoteControl takes the list of animations as a separate param. I pulled that out of the config because I already generated the animation objects. A dozen different ways I could've handled it... just the way I chose here.